hw/ip/clkmgr/doc/dv/index.md

```diff
@@ -33,34 +25,31 @@ CLKMGR testbench has been constructed based on the [CIP testbench architecture](
 ![Block diagram](tb.svg)
 
 ### Top level testbench
-Top level testbench is located at `hw/ip/clkmgr/dv/tb.sv`. It instantiates the CLKMGR DUT module `hw/ip/clkmgr/rtl/clkmgr.sv`.
+Top level testbench is located at `hw/ip/clkmgr/dv/tb.sv`. It instantiates the CLKMGR DUT module `hw/top_earlgrey/ip/clkmgr/rtl/autogen/clkmgr.sv`.
 In addition, it instantiates the following interfaces, connects them to the DUT and sets their handle into `uvm_config_db`:
 * [Clock and reset interface]({{< relref "hw/dv/sv/common_ifs" >}})
 * [TileLink host interface]({{< relref "hw/dv/sv/tl_agent/README.md" >}})
-* CLKMGR IOs
+* CLKMGR IOs: `hw/ip/clkmgr/dv/env/clkmgr_if.sv`
 * Interrupts ([`pins_if`]({{< relref "hw/dv/sv/common_ifs" >}}))
 * Alerts ([`pins_if`]({{< relref "hw/dv/sv/common_ifs" >}}))
 * Devmode ([`pins_if`]({{< relref "hw/dv/sv/common_ifs" >}}))
-* Pwrmgr interface: "hw/ip/clkmgr/dv/env/clkmgr_pwrmgr_if.sv"
-* Idle interface: "hw/ip/clkmgr/dv/env/clkmgr_idle_if.sv"
 
 ### Common DV utility components
 The following utilities provide generic helper tasks and functions to perform activities that are common across the project:
 * [dv_utils_pkg]({{< relref "hw/dv/sv/dv_utils/README.md" >}})
 * [csr_utils_pkg]({{< relref "hw/dv/sv/csr_utils/README.md" >}})
 
 ### Compile-time configurations
-[list compile time configurations, if any and what are they used for]
+No special configuration.
```

Comment:
I think you can delete this section if there is no special configs.

Reply:
Done